dashboard/src/actions/overviewActions.js

```diff
@@ -342,12 +340,16 @@ export const setRowtoEdit =
       data[rIndex].name = data[rIndex][CONSTANTS.NAME_COPY];
       data[rIndex].metadata[CONSTANTS.SERVER_DELETION] =
         data[rIndex][CONSTANTS.SERVER_DELETION_COPY];
-      data[rIndex][CONSTANTS.IS_DIRTY_NAME] = false;
-      data[rIndex][CONSTANTS.IS_DIRTY_SERVER_DELETE] = false;
+      markItemsDirty(data[rIndex], false);
       data[rIndex][CONSTANTS.NAME_VALIDATED] = CONSTANTS.SUCCESS;
     }
     dispatch(updateDatasetType(data, type));
   };
+const markItemsDirty = (data, isDirty) => {
```

Comment:
You call this twice to "mark (all) items not dirty", which is fine; but I don't think you'd want to use this to actually "mark items dirty" because it unconditionally sets *all* dirty flags to the same value. Whereas each flag will generally be set to `true` individually by the various edit handlers.

Should this be `markItemsNotDirty` or `resetDirtyFlags` or something like that, for clarity? (In which case the `isDirty` parameter wouldn't be needed.) Alternatively, I think it'd need a way to identify *which* specific flag should be *set* to mark it dirty ...

Reply:
I agree with Dave:  it would be better if this function were named something like `resetDirtyFlags()`, in which case it wouldn't need the second parameter.